std_msgs/Empty cue
---
jsk_2014_picking_challenge/QrStamp qrstamps